src/hal/display.cpp

```diff
@@ -90,11 +90,21 @@ void OswHal::increaseBrightness(uint8_t v) {
   } else {
     _brightness += v;
   }
+  #ifdef ESP32
```

Comment:
could you change this to setBrightness(_brightness)

would be less code duplication that way. thank you :)

Reply:
Yeah but its 1 more memory operation. so slower.

Reply:
But yeah if this is "cleaner" in your opinion i can change it ofcourse!

Reply:
I understand you.

My perspective was: fewer places to change stuff if we eventually use a different display.
Also 1 more memory operation versus 240Mhz on a function that currently get's called once when the watch wakes up and once when it goes back to sleep. Even gradually dimming at 100fps would still leave tons of overhead of performance...